src/info/channel.go.tmpl

```diff
@@ -11,21 +11,29 @@
 	{{ $channel = $args.Get 0 }}
 {{ end }}
 
-{{ $types := cslice "Text" "DM" "Voice" "Group DM" "Category" "News" "Store" }}
+{{ $types := sdict 
+	"0" "Text" "1" "DM" "2" "Voice" "3" "Group DM" "4" "Category" "5" "Announcement" 
+	"10" "Thread - Announcement" "11" "Thread - Public" "12" "Thread - Private" 
+	"13" "Stage" "14" "Directory" "15" "Forum" "16" "Media"
+}}
+{{ $type := $types.Get (json $channel.Type) }}
 {{ $nsfw := "No" }}
 {{ $parent := "*None set*" }}
 {{ if $channel.NSFW }} {{ $nsfw = "Yes" }} {{ end }}
-{{ with $channel.ParentID }} {{ $parent = printf "<#%d>" . }} {{ end }}
-{{ $createdAt := div $channel.ID 4194304 | add 1420070400000 | mult 1000000 | toDuration | (newDate 1970 1 1 0 0 0).Add }}
-{{ sendMessage nil (cembed
+{{ with $channel.ParentID }}
+	{{ $parent = printf "<#%d>" . }}
+	{{ if (getChannel .).IsForum }} {{ $type = "Thread - Forum" }} {{ end}}
+{{ end }}
+{{ $createdAt := snowflakeToTime $channel.ID }}
+{{ sendMessage nil ($e:=cembed
```

Comment:
Is `$e` used later?

Reply:
This was left-over from testing, sorry about that; will remove.